mm2src/coins/lightning/ln_platform.rs

```diff
@@ -128,15 +133,33 @@ async fn get_funding_tx_bytes_loop(rpc_client: &UtxoRpcClientEnum, tx_hash: H256
     }
 }
 
+pub struct LatestFees {
+    background: AtomicU64,
+    normal: AtomicU64,
+    high_priority: AtomicU64,
+}
+
+impl LatestFees {
+    #[inline]
+    fn set_background_fees(&self, fee: u64) { self.background.store(fee, Ordering::Relaxed); }
+
+    #[inline]
+    fn set_normal_fees(&self, fee: u64) { self.normal.store(fee, Ordering::Relaxed); }
+
+    #[inline]
+    fn set_high_priority_fees(&self, fee: u64) { self.high_priority.store(fee, Ordering::Relaxed); }
```

Comment:
Please consider using `Ordering::Release` memory order for these write operations.

Reply:
Done